src/scripts/main.js

```diff
@@ -18,10 +18,39 @@ const food = {
   },
 };
 
+const newElement = document.createElement('div');
+
+newElement.setAttribute('id', 'tree');
+document.body.append(newElement);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  element.append(createTreeDom(data));
+}
+
+function createTreeDom(data) {
+  if (!Object.keys(data).length) {
+    return;
```

Comment:
The `createTreeDom` function returns `undefined` when the data object is empty. This could lead to appending `undefined` to the DOM. Consider returning an empty `ul` element instead to avoid potential issues.